packages/material-ui/src/FormLabel/FormLabel.test.js

```diff
@@ -32,6 +32,7 @@ describe('<FormLabel />', () => {
 
       expect(container.querySelector('label')).to.have.text('name\u2009*');
       expect(container.querySelectorAll(`.${classes.asterisk}`)).to.have.lengthOf(1);
+      expect(container.querySelectorAll(`.${classes.asterisk}`)[0]).toBeAriaHidden();
```

Comment:
Perfect :+1: Could you include in the test description that the asterisk should be "inaccessible"?